charts/console/templates/secret.yaml

```diff
@@ -0,0 +1,42 @@
+{{- if .Values.secret.create -}}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "console.fullname" . }}
+  labels:
+  {{- include "console.labels" . | nindent 4 }}
+type: Opaque
+data:
+  # Set empty defaults, so that we can always mount them as env variable even if they are not used.
+  # For this reason we can't use `with` to change the scope.
+  # Kafka
+  kafka-sasl-password: {{ .Values.secret.kafka.saslPassword | default "" | b64enc | quote }}
+  kafka-protobuf-git-basicauth-password: {{ .Values.secret.kafka.protobufGitBasicAuthPassword | default "" | b64enc | quote }}
+  kafka-sasl-aws-msk-iam-secret-key: {{ .Values.secret.kafka.awsMskIamSecretKey | default "" | b64enc | quote }}
+  kafka-tls-ca: {{ .Values.secret.kafka.tlsCa | default "" | b64enc | quote }}
+  kafka-tls-cert: {{ .Values.secret.kafka.tlsCert | default "" | b64enc | quote }}
+  kafka-tls-key: {{ .Values.secret.kafka.tlsKey | default "" | b64enc | quote }}
+  kafka-schema-registry-password: {{ .Values.secret.kafka.schemaRegistryPassword | default "" | b64enc | quote }}
+  kafka-schemaregistry-tls-ca: {{ .Values.secret.kafka.schemaRegistryTlsCa | default "" | b64enc | quote }}
+  kafka-schemaregistry-tls-cert: {{ .Values.secret.kafka.schemaRegistryTlsCert | default "" | b64enc | quote }}
+  kafka-schemaregistry-tls-key: {{ .Values.secret.kafka.schemaRegistryTlsKey | default "" | b64enc | quote }}
+
+  # Login
+  login-jwt-secret: {{ randAlphaNum 32 | b64enc | quote }}
+  login-google-oauth-client-secret: {{ .Values.secret.login.google.clientSecret | default "" | b64enc | quote }}
+  login-google-groups-service-account.json: {{ .Values.secret.login.google.groupsServiceAccount | default "" | b64enc | quote }}
+  login-github-oauth-client-secret: {{ .Values.secret.login.github.clientSecret | default "" | b64enc | quote }}
+  login-github-personal-access-token: {{ .Values.secret.login.github.personalAccessToken | default "" | b64enc | quote }}
+  login-okta-client-secret: {{ .Values.secret.login.okta.clientSecret | default "" | b64enc | quote }}
+  login-okta-directory-api-token: {{ .Values.secret.login.okta.directoryApiToken | default "" | b64enc | quote }}
+  login-generic-oidc-client-secret: {{ .Values.secret.login.oidc.clientSecret | default "" | b64enc | quote }}
```

Comment:
This should be `login-oidc-client-secret` , was filed by a user via this PR: https://github.com/redpanda-data/console/pull/549 .

Reply:
no material changes in this migration PR